docs/server-side-search.rst

```diff
@@ -194,3 +194,7 @@ If you are using :ref:`private versions <versions:privacy levels>`,
 users will only be allowed to search projects they have permissions over.
 Authentication and authorization is done using the current session,
 or any of the valid :doc:`sharing methods </commercial/sharing>`.
+
+Cookie sessions can be used only from the same domain as the API,
```

Comment:
This needs to be explained more. What is the takeaway for a user? When should they use this URL instead? 

Presumably this should use a readthedocs.com example, where it's most necessary.

Reply:
> This needs to be explained more. What is the takeaway for a user? When should they use this URL instead?

This is under the "Authentication and authorization" section of this doc, the paragraph above explains this is for private versions. I have changed this paragraph to avoid mentioning cookies.